src/main/java/com/internetitem/logback/elasticsearch/PropertySerializer.java

```diff
@@ -0,0 +1,67 @@
+package com.internetitem.logback.elasticsearch;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.internetitem.logback.elasticsearch.util.AbstractPropertyAndEncoder;
+
+import java.io.IOException;
+
+class PropertySerializer<T> {
+    private PropertySerializer() {
+    }
+
+    private static PropertySerializer INSTANCE = null;
```

Comment:
My only question is why not just initialize here, since we are always guaranteed to need an instance (and then we can skip needing a `synchronized` block around `getInstance()`)?

Reply:
You are right, there is no need for singleton pattern on this class so there is also no need for the synchronized instantiation. I just pushed a fix for this.